projects/1.19/assets/frozen-up/frozenup/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+  "item.frozenup.truffle_muffin": "松露松饼",
+  "item.frozenup.chilloo_spawn_egg": "寒猹刷怪蛋",
+  "item.frozenup.chilloo_feather": "寒猹羽毛",
+  "item.frozenup.empty_mug": "空马克杯",
+  "item.frozenup.mug_of_chocolate_milk": "杯装热巧克力",
+  "item.frozenup.mug_of_truffle_hot_chocolate": "杯装松露热巧克力",
+  "item.frozenup.mug_of_milk": "杯装奶",
+ 
+  "block.frozenup.compacted_snow_bricks": "压实雪砖",
+  "block.frozenup.compacted_snow_brick_wall": "压实雪砖墙",
+  "block.frozenup.compacted_snow_brick_slab": "压实雪砖台阶",
+  "block.frozenup.compacted_snow_brick_stairs": "压实雪砖楼梯",
+  "block.frozenup.compacted_snow_foundation": "压实雪制地基",
+ 
+  "block.frozenup.compacted_snow_brick_vertical_slab": "压实雪砖竖直台阶",
+  "block.frozenup.cut_ice_vertical_slab": "切冰竖直台阶",
+  "block.frozenup.cut_ice_cube_vertical_slab": "切冰块竖直台阶",
+ 
+  "block.frozenup.cut_ice": "切冰",
+  "block.frozenup.cut_ice_stairs": "切冰楼梯",
+  "block.frozenup.cut_ice_slab": "切冰台阶",
+  "block.frozenup.cut_ice_cubes": "切冰块",
+  "block.frozenup.cut_ice_cube_stairs": "切冰块楼梯",
+  "block.frozenup.cut_ice_cube_slab": "切冰砖台阶",
+  "block.frozenup.cut_ice_cube_wall": "切冰块墙",
+  "block.frozenup.chilloo_feather_block": "寒猹羽毛块",
+  "block.frozenup.chilloo_feather_block_carpet": "寒猹羽毛地毯",
+  "block.frozenup.truffle_cake": "松露巧克力蛋糕",
+  "block.frozenup.chilloo_feather_lamp": "寒猹羽毛灯",
+  "block.frozenup.empty_mug": "空马克杯",
+  "block.frozenup.mug_of_milk": "杯装奶",
+  "block.frozenup.mug_of_chocolate_milk": "杯装热巧克力",
+  "block.frozenup.mug_of_truffle_hot_chocolate": "杯装松露热巧克力",
+ 
+  "entity.frozenup.reindeer": "驯鹿",
+  "entity.frozenup.penguin": "企鹅",
+  "entity.frozenup.chilloo": "寒猹",
+   
+  "subtitles.entity.frozenup.reindeer.gallop": "驯鹿：肌腱咔哒声",
+  "subtitles.entity.frozenup.reindeer.breathe": "驯鹿：喘息",
+  "subtitles.entity.frozenup.reindeer.idle": "驯鹿：呼噜声",
+  "subtitles.entity.frozenup.reindeer.hurt": "驯鹿：受伤",
+  "subtitles.entity.frozenup.reindeer.death": "驯鹿：死亡",
+  "subtitles.entity.frozenup.reindeer.angry": "驯鹿：愤怒",
+  "subtitles.entity.frozenup.reindeer.eat": "驯鹿：大声咀嚼",
+  "subtitles.entity.frozenup.reindeer.jump": "驯鹿：跳跃",
+  "subtitles.entity.frozenup.reindeer.cloud_jump": "驯鹿：云跃",
+  "subtitles.entity.frozenup.penguin.idle": "企鹅：尖锐地叫",
+  "subtitles.entity.frozenup.penguin.hurt": "企鹅：受伤",
+  "subtitles.entity.frozenup.penguin.death": "企鹅：死亡",
+  "subtitles.entity.frozenup.penguin.crack": "企鹅：试图挣脱",
+  "subtitles.entity.frozenup.penguin.hatch": "企鹅：破壳而出",
+  "subtitles.entity.frozenup.penguin.slide": "企鹅：滑行",
+  "subtitles.item.frozenup.pinecone.eat": "咀嚼",
+  "subtitles.frozenup.entity.chilloo.ambient": "寒猹：啾啾叫",
+  "subtitles.frozenup.entity.chilloo.hurt": "寒猹：受伤",
+  "subtitles.frozenup.entity.chilloo.death": "寒猹：死亡",
+  "subtitles.frozenup.entity.chilloo.eat": "寒猹：咀嚼",
+  "subtitles.frozenup.entity.chilloo.spit": "寒猹：吐",
+  "subtitles.frozenup.block.chilloo_feather_lamp.toggle_on": "开灯",
+  "subtitles.frozenup.block.chilloo_feather_lamp.toggle_off": "关灯",
+  "subtitles.item.block.mug.drink": "喝"

```

```suggestion
  "subtitles.frozenup.block.chilloo_feather_lamp.toggle_on": "灯：打开",
  "subtitles.frozenup.block.chilloo_feather_lamp.toggle_off": "灯：关闭",
  "subtitles.item.block.mug.drink": "饮用"
```
当然最后一项看声音整成`啜饮`应该也没有问题
https://minecraft.fandom.com/zh/wiki/%E5%AD%97%E5%B9%95

---

这一处开灯/关灯是一开始一位维护者提出，暂时保留
“喝”这一项我觉得“啜饮”更好
其他建议我都赞同并修改完毕
谢谢你的建议！

---

字幕采用 `发声主体：声音` 格式，这里的建议更好。

`灯：开灯` 显然不行

---

如果使用灯:打开或者灯:被打开呢
参见我的世界wiki“活板门:打开”
[]()https://minecraft.fandom.com/zh/wiki/%E5%AD%97%E5%B9%95

---

> 如果使用灯:打开或者灯:被打开呢 参见我的世界wiki“活板门:打开” https://minecraft.fandom.com/zh/wiki/%E5%AD%97%E5%B9%95

那就是chuijk 的建议